packages/react/src/components/ActionMenu/ActionMenu.test.tsx

```diff
@@ -417,6 +418,17 @@ test('should support ref prop', () => {
   expect(ref.current).toEqual(screen.getByTestId('actionmenu'));
 });
 
+test('should support portal prop', async () => {
```

Comment:
We should add a test here to ensure when the `ActionMenu` is placed in some parent element that getting the parent of `ActionMenu` is the element we expect

Reply:
This already does:

```ts
const actionMenuInPortal = await findByTestId(portal, 'actionmenu');
```

`findByTestId` passes in `portal` to check that the action menu is present.